src/Resources/app/storefront/src/checkout/confirm-order.plugin.js

```diff
@@ -92,7 +92,7 @@ export default class ConfirmOrderPlugin extends Plugin {
             // TODO error handling
             return;
         }
-        window.orderId = !!order.data ? order.data.id : order.id;
+        window.orderId = order.id;
```

Comment:
This wasn't working for me a couple of weeks back and that's why I changed it here https://github.com/Adyen/adyen-shopware6/pull/125/files

But now with this PR I cannot reproduce the issue, so maybe it was my setup. Was this causing problems for you and that's why you undid it @peterojo?

Reply:
Oh you're right @acampos1916,  I thought I was the one who I added this when I [switched SalesChannel API for store API](https://github.com/Adyen/adyen-shopware6/pull/122/files#diff-ebd93096f9268cf9ecee4a9e5b146d5968c0fdcb0234dfd4bd284b8fcfd05403L95), I think it's okay to remove this check. The response from the `store-api` endpoint removes the `data` property and just returns the order in the root. 
So it's now `{id:xxxx,orderNumber:"yyyy",...}` instead of `{data:{id:xxxx,orderNumber:"yyyy"...}}`
So maybe you didn't yet have that change when you tested it?

Reply:
Exactly! Thanks @peterojo